content/c-sharp/concepts/loops/loops.md

```diff
@@ -0,0 +1,86 @@
+---
+Title: 'Loops'
+Description: 'A loop is a logical structure that allows for the repeated execution of multiple iterations of a block of code in C#'
+Subjects:
+  - 'Computer Science'
+  - 'Code Foundations'
+Tags:
+  - 'Loops'
+CatalogContent:
+  - 'learn-c-sharp'
+  - 'paths/computer-science'
+---
+
+A **loop** is a logical structure used in [C#](https://www.codecademy.com/resources/docs/c-sharp) to repeatedly execute a block of code. This repetition can either iterate an exact number of times (definite) or continuously until a condition is met (indefinite).
+
+## For Loop
+
+In C# the `for` loop uses definite repetition to run a block of code a specified number of times and consists of three statements separated by semicolons.
+
+```cs
+for (int i = 5; i > 0; i--)
+{
+  //repeated code here
+}
+```
+Statement 1 executes only once, before the code block runs for the first time, and is commonly used to set the initial value of a variable.
+
+Statement 2 is a conditional statement that must return `true` for the code block to execute.
+
+Statement 3 runs after each repetition of the loop and is typically used to alter the value of the variable set in Statement 1.
+
+```codebyte/csharp
```

Comment:
The codebytes as written will not run because they are missing a class with a 'Main()' method. I'd remove them.

Reply:
Done